src/providers/twitch/TwitchMessageBuilder.cpp

```diff
@@ -795,6 +795,12 @@ void TwitchMessageBuilder::runIgnoreReplaces(
         {
             continue;
         }
+        const auto &user = phrase.getUser();
+        if (user.isEmpty() &&
```

Comment:
Shouldn't this be 
```suggestion
        if (!user.isEmpty() &&
```
So it's the same as before?

Reply:
Also you can press commit suggestion here on github and it'll automatically do the work for you if you agree with the change.